plugins/out_s3/s3.c

```diff
@@ -976,8 +979,11 @@ static int upload_data(struct flb_s3 *ctx, struct s3_file *chunk,
     int size_check = FLB_FALSE;
     int part_num_check = FLB_FALSE;
     int timeout_check = FLB_FALSE;
+    ctx->is_compressed=FLB_FALSE;
```

Comment:
I don't think we need to put this line here. It is not a definition

Reply:
+1

What is the purpose for this variable on the context? AFAICT, its not needed in the logic.

Reply:
removed